platform/security/platform-security-core-api/src/main/java/ddf/security/principal/AnonymousPrincipal.java

```diff
@@ -18,11 +18,11 @@
 import org.apache.commons.lang.StringUtils;
 
 /**
- * Principal that designates a {@link ddf.security.Subject} as anonymous
+ * Principal that designates a {@link ddf.security.Subject} as guest
  */
 public class AnonymousPrincipal implements Principal {
```

Comment:
Is there a reason why we did not rename classes to use guest instead of anonymous?